backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -190,7 +191,7 @@ public void addQueue(PresetQueue queue) {
       statement = connection.prepareStatement(sql);
       statement.setInt(1, queue.getID());
       statement.setString(2, queue.getName());
-      statement.executeUpdate();
+      statement.executeUpdate(sql);
```

Comment:
When you execute a prepared statement, you do not need to re-assign the query in de execution.